platform/ui/jest.config.js

```diff
@@ -0,0 +1,13 @@
+const base = require('../../jest.config.base.js');
```

Comment:
why we are adding this?

Reply:
I don't think the unit tests that were added were running until I added this file. I basically copied the file from core. I can try to remove it and see if the unit tests get picked up.

Reply:
I just confirmed that without this file the useSessionStorage.test.js will NOT get run.